plugin.rb

```diff
@@ -18,3 +18,9 @@
 require_relative "lib/linked_in_authenticator"
 
 auth_provider authenticator: ::LinkedInAuthenticator.new, icon: "fab-linkedin"
+
+after_initialize do
+  AdminDashboardData.add_problem_check do
+    "The discourse-linkedin-auth plugin is no longer supported, and LinkedIn OpenID Connect support has been added to Discourse core. You are recommended to update the authentication method in your LinkedIn app. Please see https://meta.discourse.org/t/discourse-linkedin-authentication/46818 for more information."
+  end
```

Comment:
What do admins need to do to clear the warning? Looks like right now they have to totally uninstall the plugin?

That's not possible for admins to do on our hosting. So perhaps we need to make this problem check conditional based on the value of the 'enabled' site setting?

Reply:
Good catch! Added the conditional. 👍